azure-keyvault/azure/keyvault/vault_client.py

```diff
@@ -5,6 +5,7 @@
 class VaultClient(object):
 
     def __init__(self, vault_url, credentials, config=None, **kwargs):
+        self.vault_url = vault_url
```

Comment:
>self.vault_url = vault_url [](start = 8, length = 26)

I would make this _vault_url to discourage people from doing things like setting this on existing clients